src/components/editors/layoutEditor.css

```diff
@@ -0,0 +1,12 @@
+.MuiGrid-item {
```

Comment:
Namespace the class to LayoutEditor.js, otherwise you are affecting everything globally

Reply:
e.g. `.layout-editor .MuiGrid-item`